cases/closeSessionOnRecreateApplication/config.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+return [
+    'id' => 'CloseSessionOnRecreateApplication',
+    'basePath' => __DIR__,
+    'controllerNamespace' => 'app\closeSessionOnRecreateApplication\controllers',
+    'components' => [
+        'request' => [
+            'enableCsrfValidation' => false,
```

Comment:
This seems irrelevant since you are only using GET requests.

Reply:
Fixed, thanks